drivers/storage/portworx/components_test.go

```diff
@@ -11564,7 +11564,10 @@ func TestTelemetryEnableAndDisable(t *testing.T) {
 	require.NoError(t, err)
 
 	expectedDeployment := testutil.GetExpectedDeployment(t, "metricsCollectorDeployment.yaml")
-	require.True(t, util.DeploymentDeepEqual(expectedDeployment, deployment))
+	// Revise unrelated fields so we can compare.
+	expectedDeployment.Spec.Template.Spec.DeprecatedServiceAccount = ""
```

Comment:
Instead of setting the deprecated service account empty here, let's just remove it from the test spec.

Reply:
This comment doesn't seem to be addressed.

Reply:
missed this one, it is not in test spec. it is deprecated but gets set even if it is not in test spec.